integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchmarkRunner.scala

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids.tests.common
+
+import com.nvidia.spark.rapids.tests.tpcds.TpcdsLikeBench
+import com.nvidia.spark.rapids.tests.tpch.TpchLikeBench
+import com.nvidia.spark.rapids.tests.tpcxbb.TpcxbbLikeBench
+import org.rogach.scallop.ScallopConf
+
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+/**
+ * The BenchmarkRunner can be submitted using spark-submit to run any of the TPC-* benchmarks.
+ */
+object BenchmarkRunner {
+
+  def main(args: Array[String]): Unit = {
+    val conf = new BenchmarkConf(args)
+
+    val bench: BenchmarkSuite = conf.benchmark() match {
+      case "tpcds" => TpcdsLikeBench
+      case "tpch" => TpchLikeBench
+      case "tpcxbb" => TpcxbbLikeBench
+      case other => throw new IllegalArgumentException(s"Invalid benchmark name: $other")
```

Comment:
I suggest building a `Map` of names to benchmark classes.  Then we can emit a nicer error message on an unrecognized benchmark name by dumping the keys to the map showing what is supported.